core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java

```diff
@@ -466,8 +466,10 @@ public void provideOutcome(AppCreator ctx) throws AppCreatorException {
     private boolean isThisGraalVMRCObsolete() {
```

Comment:
I have a small question @Sanne regarding this method. It is commented with `//FIXME remove after transition period` ? A quick search indicates that it's used only once in this file line `267` assigned to a boolean that is never used. What was the transition period? Are we still in it?

Reply:
Yes we still are in a "transition period".

By that I mean that we'll be following each latest GraalVM update. Eventually we'll have it depend on stable tags, then we'll have to revisit how we do this.

Reply:
Thanks for the response.